features/steps.js

```diff
@@ -96,12 +96,25 @@ module.exports = function () {
   })
 
   this.Then(/^the file "([^"]*)" should have been uploaded to the container$/, function (name) {
+    if (!this.fileUploads) {
+      throw new Error('there was no files uploaded to the server')
+    }
     var file = find(this.fileUploads, hasProps({ name: name }))
     if (!file) {
       throw new Error('there was no file named "' + name + '" uploaded to the server')
     }
   })
 
+  this.Then(/^the path in the container should be "([^"]*)"$/, function (path) {
+    if (!this.fileUploads) {
+      throw new Error('there was no files uploaded to the server')
+    }
+    var file = find(this.fileUploads, hasProps({ path: path }))
+    if (!file) {
+      throw new Error('there was no file with path "' + path + '" uploaded to the server')
+    }
+  })
```

Comment:
merge this with the above function. I think what you want to end up with (because it reads better in the cucumber test)

```js
  // adjust regex here
  this.Then(/^the file "([^"]*)" should have been uploaded to "([^"]*)"$/, function (name, path) {
    if (!this.fileUploads) {
      throw new Error('there was no files uploaded to the server')
    }
    var file = find(this.fileUploads, hasProps({ name: name, path: path })) // add path here
    if (!file) {
      throw new Error('there was no file named "' + name + '" uploaded to the server')
    }
    // or you can find the file w/ just the name and then verify that the path is correct. up to you :)
  })
```